src/JsonPointer/JsonPointer.cs

```diff
@@ -322,6 +352,9 @@ public JsonPointer Combine(JsonPointer other)
 	/// <returns>A new pointer.</returns>
 	public JsonPointer Combine(params PointerSegment[] additionalSegments)
 	{
+#if NET9_0_OR_GREATER
+		return Combine(additionalSegments.AsSpan());
```

Comment:
Same here.  The array is already allocated.  Calling the span version doesn't give you anything.

Reply:
See above